### PR TITLE
add empty prettierrc.json and add prettierignore

### LINKED ISSUE #3: Make a Prettier Config file
Make a prettier configuration file so our code can follow the same indenting, spacing etc. 

https://prettier.io/docs/en/configuration.html

### PR DESCRIPTION
**Pull request by Jaqito**
closes #3 - Add Prettier Config

PR Adds an empty prettierrc which indicates to VSCode to use the default prettier config. Also adds a prettierignore to ignore certain files that are not checked into Github.

### CODE REVIEW

File: .fx/subscriptionInfo.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "subscriptionId": "0f16d84b-3c5e-4cfa-be32-a78c3329897b",
+  "subscriptionName": "Azure for Students",
+  "tenantId": "ce6d05e1-3c5e-4d62-87a8-4c4a2713c113"
+}
```
Comment (rorochaudhary) [inline .fx/subscriptionInfo.json]: Was this an intended change for the PR, or is this a side effect of Azure doing it's thing?

Comment (Jaqito) [reply .fx/subscriptionInfo.json]: Azure keeps adding it. It's actually from the other branch but everytime I open vscode it adds it again

Comment (Jaqito) [reply .fx/subscriptionInfo.json]: Good point though I'll delete  from this branch

Comment (Jaqito) [reply .fx/subscriptionInfo.json]: deleted.

**Review by rorochaudhary — APPROVED** [review]
Thanks for setting this up! I was eyeing this PR but got caught up in other things and it fell to the wayside.

**Pull request merged**